crates/consensus/consensus/src/lib.rs

```diff
@@ -451,6 +451,11 @@ pub enum ConsensusError {
         /// The block's timestamp.
         timestamp: u64,
     },
+    /// Custom error
+    // todo: remove in favour of AT Consensus::Error, so OpConsensusError can wrap ConsensusError
```

Comment:
I think this makes sense,

this tracks with what we want to do for the configure evm stuff as well